ngraph/frontend/frontend_manager/include/frontend_manager/frontend.hpp

```diff
@@ -26,43 +27,28 @@ namespace ngraph
 
             virtual ~FrontEnd();
 
-            /// \brief Loads an input model by specified model file path
-            /// If model is stored in several files (e.g. model topology and model weights) -
-            /// frontend implementation is responsible to handle this case, generally frontend may
-            /// retrieve other file names from main file
-            /// \param path Main model file path
+            /// \brief Validates if FrontEnd can recognize model with parameters specified.
+            /// Same parameters should be used to load model.
+            /// \param vars Any number of parameters of any type
+            /// \return true if model recognized, false - otherwise
+            template <typename... Types>
+            bool supported(const Types&... vars) const
```

Comment:
Why do we need this method? I don't see that we use this method in reader.

Reply:
This method is used by FrontendManager in load_by_lodel function